doc/examples/plot_seam_carving.py

```diff
@@ -0,0 +1,94 @@
+"""
+============
+Seam Carving
+============
+
+This example demonstrates how images can be resized using seam carving [1]_.
+Resizing often distorts contents in the image. Seam carving tries to resize
```

Comment:
This or something like it would be a bit clearer:

"Resizing to a new aspect ratio distorts image contents. Seam carving attempts to resize \*without* distortion, by removing regions of an image which are visually similar."